src/doc/src/reference/unstable.md

```diff
@@ -396,6 +397,32 @@ It's possible to update `my-dependency` to a pre-release with `update -Zunstable
 This is because `0.1.2-pre.0` is considered compatible with `0.1.1`.
 It would not be possible to upgrade to `0.2.0-pre.0` from `0.1.1` in the same way.
 
+## sbom
```

Comment:
Should we document the schema in here?  yes, the RFC will have it but that will then be a snapshot in time and eventually we'll want to have it documented.